src/index.html

```diff
@@ -0,0 +1,69 @@
+<html>
+  <head>
+    <link rel="stylesheet" href="css/vote-CSS.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap'"
+    />
+  </head>
+  <body>
+    <div class="test_box">
+      <div class="u-layout">
+        <img class="u-layout" src="c:\Pics\logo.png" alt="logo" />
+      </div>
+
+      <div class="u-layout">
+        <h1>Welcome Back</h1>
+      </div>
+      <div class="u-layout">
+        <img id="google_logo" src="c:\Pics\google.png" alt="google_logo" />
+      </div>
+      <div class="u-layout">
+        <h2 id="header">OR LOGIN WITH EMAIL</h2>
+      </div>
+      <form>
+        <div class="u-form-sizes">
+          <input
+            class="c-form"
```

Comment:
I think the name of this class should be `c-input` as that is what it is. It's not a form, just an element inside a form 👍